src/main/kotlin/com/exactpro/th2/common/metrics/SessionAliasWithDirectionAndMessageType.kt

```diff
@@ -0,0 +1,29 @@
+package com.exactpro.th2.common.metrics
```

Comment:
The same recommend as for the SessionAliasWithDirection file

Reply:
done